packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -134,29 +171,51 @@ export const prepareLocalChainAccountKit = (
         async makeDelegateInvitation(validatorAddress, ertpAmount) {
           trace('makeDelegateInvitation', validatorAddress, ertpAmount);
 
+          return zcf.makeInvitation(async seat => {
+            // TODO should it allow delegating more BLD?
+            seat.exit();
+            return this.facets.holder.delegate(validatorAddress, ertpAmount);
+          }, 'Delegate');
+        },
+        /**
+         *
+         * @param {string} validatorAddress
+         * @param {Amount<'nat'>} ertpAmount
+         */
+        async undelegate(validatorAddress, ertpAmount) {
           // TODO #9211 lookup denom from brand
           const amount = {
             amount: String(ertpAmount.value),
             denom: 'ubld',
           };
+          const { account: lca } = this.state;
+          trace('lca', lca);
+          const delegatorAddress = await E(lca).getAddress();
+          trace('delegatorAddress', delegatorAddress);
+          const [result] = await E(lca).executeTx([
+            typedJson('/cosmos.staking.v1beta1.MsgUndelegate', {
+              amount,
+              validatorAddress,
+              delegatorAddress,
+            }),
+          ]);
+          trace('got result', result);
```

Comment:
This is supposed to wait until the unbonding period is over, isn't it?

Reply:
API: 

https://github.com/Agoric/agoric-sdk/blob/e7745e63ea21cbd29b72d01aff77dca75e44e70a/packages/orchestration/src/cosmos-api.ts#L152-L158

precedent:

https://github.com/Agoric/agoric-sdk/blob/e7745e63ea21cbd29b72d01aff77dca75e44e70a/packages/orchestration/src/exos/stakingAccountKit.js#L410-L413